Harmonizing wording and adding VSS to IaaS

floschmsft opened the pull request:
Azure Backup Sever was listing "App-aware snapshots (VSS)", DPM "Application-aware snapshots (VSS)". Harmonized it for better readability.

Azure IaaS VM Backup is also application consistent (at least it lists it in the portal as "application consistent backup"). maybe the entire article should be proof-read again to reflect the current featureset of the options.

azurecla commented:
Hi __@abaranch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (abaranch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

alancameronwills commented:
editors, please pull

davidwrede commented:
@abaranch, if this PR is still relevant, please resolve any merge conflicts.

Thanks,
Dave

abaranch commented:
Abandoning this and recreating a new one #5243 so I do not have to resolve conflicts.

floschmsft committed:
Harmonizing wording and adding VSS to IaaS

Azure Backup Sever was listing "App-aware snapshots (VSS)", DPM "Application-aware snapshots (VSS)". Harmonized it for better readability.

Azure IaaS VM Backup is also application consistent (at least it lists it in the portal as "application consistent backup"). maybe the entire article should be proof-read again to reflect the current featureset of the options.

GitHubber17 committed:
Update backup-introduction-to-azure-backup.md
